src/lib/cache.coffee

```diff
@@ -0,0 +1,54 @@
+###
+
+dnschain
+http://dnschain.net
+
+Copyright (c) 2014 okTurtles Foundation
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+###
+
+redis = require 'redis'
+
+module.exports = (dnschain) ->
+    # expose these into our namespace
+    for k of dnschain.globals
+        eval "var #{k} = dnschain.globals.#{k};"
+
+    class ResolverCache
+        constructor: (@dnschain) ->
+            @log = gNewLogger 'CACHE'
+            @log.info "Loading Redis Cache..."
+            if gConf.chains.dnschain.get('redis:enabled')
```

Comment:
Could you go through your code and replace all instances of `gConf.chains.dnschain.get/set` with `gConf.get/set`?